### DISCUSSION TITLE
Cannot delete servers from the Server list

### DISCUSSION THREAD

**Discussion by ellory-yu** [codefori/vscode-ibmi#1661] [Q&A] [answered]
👉🏻 I am trying to remove old server connections in the Server list.  Every time I right click on any of the sever in the list, I do not get the pop-up menu with a Delete option.  Is this a bug or is there some other ways to delete the old server from the list.

⚠️ **REMOVE THIS LINE AND ANY SENSITIVE INFORMATION BELOW!** ⚠️

|Context|Version|
|-|-|
|Code for IBM i version|2.4.0|
|Visual Studio Code version|1.84.2|
|Operating System|linux_x64|

<details>
<summary>Active extensions</summary>

```
Code for IBM i Walkthroughs (vscode-ibmi-walkthroughs): 0.4.0
Emmet (emmet): 1.0.0
Git (git): 1.0.0
Git Base (git-base): 1.0.0
GitHub Authentication (github-authentication): 0.0.2
GitHub Repositories (remotehub): 0.62.0
JSON Language Features (json-language-features): 1.0.0
Merge Conflict (merge-conflict): 1.0.0
OpenAPI (Swagger) Editor (vscode-openapi): 4.18.6
Remote Repositories (remote-repositories): 0.40.0
TypeScript and JavaScript Language Features (typescript-language-features): 1.0.0
Vim (vim): 1.27.0
YAML (vscode-yaml): 1.14.0
```
</details>

<hr />

**_Not connected_** 🔌

**Comment by worksofliam** [comment]
I see the right click option to delete a connection/server. Do you not see that?

<img width="475" alt="image" src="https://github.com/codefori/vscode-ibmi/assets/3708366/c204902a-2537-48f1-adaa-3f05319ec02e">

**Comment by ellory-yu** [comment]
No I’m not seeing it on Mint Linux.  I do see it running VS Code on Windows 10 Pro.  On Mint Linux, every time I select any of the servers by right clicking on it, it only executes the connection.

**Reply by sebjulliand** [reply]
Are you using Mint on a laptop with a touchpad? Does the right click "misbehave" the same way all over VSCode? For example, do you see the right-click menu when you right-click on a regular file in the file explorer?

**Reply by ellory-yu** [reply, marked as answer]
Yes, I am using Mint on a laptop with a touchpad.  I just tried your question. The right click "misbehave" the same way.  So this is not a VSCode issue.  I'm not sure why the touch pad right click is misbehaving on Mint.  Thank you for your help.

**Reply by sebjulliand** [reply]
I saw a few other people complaining about the same issue, that's why I asked. I guess you can find a solution on a forum 😃
